src/types/Constants.ts

```diff
@@ -162,7 +162,7 @@ export enum ROLES {
   SETUP_CLIENT = 'setup_client',
   APPSTORE_VIEW = 'view_apps',
   APPSTORE_VIEW_SERVICES = 'view_service_marketplace',
-  SUBSCRIBE_SERVICE_MARKETPLACE = 'subscribe_service_offering',
+  SUBSCRIBE_SERVICE_MARKETPLACE = 'subscribe_service',
```

Comment:
@nidhigarg-bmw thats missing in the changeLog, please add

Reply:
@jjeroch Just added